libcxx/docs/ReleaseNotes/21.rst

```diff
@@ -42,6 +42,7 @@ Implemented Papers
 - P0767R1: Deprecate POD (`Github <https://github.com/llvm/llvm-project/issues/104013>`__)
 - P1361R2: Integration of chrono with text formatting (`Github <https://github.com/llvm/llvm-project/issues/100014>`__)
 - P2255R2: A type trait to detect reference binding to temporary (implemented the type traits only) (`Github <https://github.com/llvm/llvm-project/issues/105180>`__)
+- P2322R6: ``ranges::fold`` (implemented ``ranges::fold_left_first`` and ``ranges::fold_left_first_with_iter``) (`Github <https://github.com/llvm/llvm-project/issues/105208>`__)
```

Comment:
We only mention papers in the release notes when they are complete.

Reply:
Reverted change in this line.